src/main/java/cmpe/dos/entity/Rating.java

```diff
@@ -0,0 +1,86 @@
+package cmpe.dos.entity;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "Rating")
```

Comment:
Table name should be"RATING" (uppercase). There's no error yet, because Hibernate isn't looking for the table. We will merge when rating service and controller are also checked in.